app/experimenter/nimbus-ui/src/components/PageRequestReview/FormLaunchPreviewToReview.tsx

```diff
@@ -19,40 +20,54 @@ const FormLaunchPreviewToReview = ({
   const [allBoxesChecked, setAllBoxesChecked] = useState(false);
 
   return (
-    <Alert variant="warning">
-      <Form className="text-body">
-        <p className="my-1">
-          This experiment is currently <strong>live for testing</strong>, but
-          you will need to let QA know in your PI request. When you have
-          recieved a sign-off, click “Request launch” to launch the experiment.
+    <>
+      <Alert
+        data-testid="submit-success"
+        variant="success"
+        className="bg-transparent text-success"
+      >
+        <p className="my-1" data-testid="in-preview-label">
+          <Check className="align-top" /> All set! Your experiment is in Preview
+          mode and you can test it now.
         </p>
+      </Alert>
 
-        <FormLaunchConfirmationCheckboxes onChange={setAllBoxesChecked} />
+      <Alert variant="warning">
+        <Form className="text-body">
+          <p className="my-1">
+            This experiment is currently <strong>live for testing</strong>, but
+            you will need to let QA know in your PI request. When you have
+            recieved a sign-off, click “Request launch” to launch the
+            experiment.
+          </p>
 
-        <div className="d-flex bd-highlight">
-          <div className="py-2">
-            <button
-              data-testid="launch-preview-to-review"
-              type="button"
-              className="mr-3 btn btn-primary"
-              disabled={isLoading || !allBoxesChecked}
-              onClick={onSubmit}
-            >
-              Request Launch
-            </button>
-            <button
-              data-testid="launch-preview-to-draft"
-              type="button"
-              className="btn btn-secondary"
-              disabled={isLoading}
-              onClick={onBackToDraft}
-            >
-              Go Back to Draft
-            </button>
+          <FormLaunchConfirmationCheckboxes onChange={setAllBoxesChecked} />
+
+          <div className="d-flex bd-highlight">
+            <div className="py-2">
+              <button
+                data-testid="launch-preview-to-review"
+                type="button"
+                className="mr-3 btn btn-primary"
+                disabled={isLoading || !allBoxesChecked}
+                onClick={onSubmit}
+              >
+                Request Launch
+              </button>
+              <button
+                data-testid="launch-preview-to-draft"
+                type="button"
+                className="btn btn-secondary"
+                disabled={isLoading}
+                onClick={onBackToDraft}
+              >
+                Go Back to Draft
+              </button>
+            </div>
           </div>
-        </div>
-      </Form>
-    </Alert>
+        </Form>
+      </Alert>
+    </>
```

Comment:
Would it make more sense for it to live next to [this fella](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx#L117-L128)?

Reply:
I thought maybe no, since it only shows up with this form and we might need to move where this form lives anyway